### DISCUSSION TITLE
Can't run lf after compiling

### DISCUSSION THREAD

**Discussion by Archie-2021** [gokcehan/lf#907] [Q&A] [answered]
I have installed Go (v1.19) and used `env CGO_ENABLED=0 go install -ldflags="-s -w" github.com/gokcehan/lf@latest` command to compiled it without any error. but i can't access this app using `lf`. can someone help me? I am new to Linux and can't find a solution, I ma on Linux Mint 20.3.

**Comment by lahwaacz** [comment]
You need to install `lf` into a directory which is in your `$PATH` or change the environment variable accordingly. See https://forums.linuxmint.com/viewtopic.php?t=180380

**Reply by Archie-2021** [reply]
> You need to install `lf` into a directory which is in your `$PATH` or change the environment variable accordingly. See https://forums.linuxmint.com/viewtopic.php?t=180380

I can test that but don't know which bin folder it is supposed to be installed to. the name `lf` is to generic to search. also i thought they are going to mention adding export to PATH in bashrc if it was needed

**Comment by piskunovim** [comment, marked as answer]
Hey there! I know this question was raised about 2 years ago, but I figured it might still be helpful for anyone stumbling upon this issue today. 

The installation location of Go binaries depends on your environment settings:

1. If you have set the GOBIN environment variable, it will be installed there.
2. If GOBIN is not set, but GOPATH is set, it will be installed in $GOPATH/bin.
3. If neither GOBIN nor GOPATH are set, it will default to $HOME/go/bin

Just remember to add whichever folder you're using to your PATH, and you'll be ready to Go (pun intended)! 😉 
Don't let those binaries play hide and seek with you! 🚀

**Reply by CatsDeservePets** [reply]
Thank you for giving this helpful reply even 2 years later.
